Week2/homework/map-filter.js

```diff
@@ -1,8 +1,8 @@
 'use strict';
 
 function doubleOddNumbers(numbers) {
-  // Replace this comment and the next line with your code
-  console.log(numbers);
+  const arr = numbers.filter(num => num % 2 !== 0).map(num => num * 2);
+  return arr;
```

Comment:
This is only my personal opinion but you can also combine these two statements and simply have
```
return numbers.filter(num => num % 2 !== 0).map(num => num * 2);
```